eng/Packages.props

```diff
@@ -18,6 +18,7 @@
     <PackageReference Update="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.16.30" />
     <PackageReference Update="Microsoft.Win32.Registry" Version="4.3.0" />
     <PackageReference Update="NuGet.Build.Tasks" Version="$(NuGetBuildTasksVersion)" />
+    <PackageReference Update="NuGet.Frameworks" Version="$(NuGetBuildTasksVersion)" />
```

Comment:
Is this already in VS? I imagine so, but I haven't checked.

Reply:
Can you clarify what you mean? AFAIK MSBuild doesn't take a dependency on Nuget right now, this will introduce that.

Reply:
I found it [here](https://dev.azure.com/devdiv/DevDiv/_git/VS?path=%2Fsrc%2FProductData%2FEngineering%2FExternalWhiteList.csv), so I think it's ok. It might be good to merge this into an `exp/` branch before we merge it into master to make sure RPS doesn't complain about adding an assembly.

This was my issue with the solution filters—nothing in VS had used System.Text.Json previously, so I needed to introduce it, which also required updating the versions of other assemblies, which entangled my change with work Roslyn had to do, which meant that it sat forever in limbo. I don't think you'll hit that problem, but it would be good to make sure.

Reply:
Yes, we'll definitely have VS Insertion Fun™️ with this one. Especially because I think NuGet places itself in different relative locations between .NET SDK and VS. Let's punt this for now and try to sync on it when we get a chance, though.

Reply:
A quick and easy test: can you use `scripts\Deploy-MSBuild.ps1` to deploy the output of on top of a VS installation (sacrificial! don't use your primary VS). Does that work?

Reply:
And while you're at it, you can try it with your SDK folder, adding `-runtime Core`.

Reply:
No, didn't work. I'm getting `error MSB4184: The expression "[MSBuild]::GetTargetFrameworkIdentifier(netcoreapp3.1)" cannot be evaluated. Could not load file or assembly 'NuGet.Frameworks, Version=5.4.0.2, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified.`

When I try to add the `-runtime Core` option VS can't load projects with the error `Could not load file or assembly 'Microsoft.VisualStudio.VC.RecommendationProvider, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.`.

Reply:
`-runtime Core` has to go into your sdk folder and be called through `dotnet build` rather than into your VS folder, so something like `C:\Program Files\dotnet\sdk\3.1.200-preview-015002`.

Reply:
microsoft/msbuild#5245: we should have docs on the script, it's not obvious

Unfortunately, that's the failure I expected from the VS scenario. Let's sync tomorrow about causes/possible solutions.

Reply:
Include me?
And do you want me to send you the video about adding an assembly to VS? I imagine this is quite different, since the assembly is coming from a different source, and VS already seems to have some basic conception of it, but it might still be helpful.

Reply:
Sure, if you think it's helpful you can send it over.